doc/examples/reconst_csd.py

```diff
@@ -98,18 +101,17 @@
 fvtk.rm(ren, response_actor)
 
 """
-However, using an FA threshold is not a very robust method. It is dependent on
-the dataset (non-informed used subjectivity), and still depends on the
-diffusion tensor (FA and first eigenvector), which has low accuracy at high
-b-value. Alternatively, one can calibrate the response function directly from
-the data according to [Tax2014]_. 
+Depending on the dataset, FA threshold may not be the best way to divine the
```

Comment:
Keep it simple. This use of the word is very rare afaik. 
Even guess is too vague here. 

I would say: find the best possible response function.
Your call!